datadog-crashtracker/src/collector/emitters.rs

```diff
@@ -47,6 +47,13 @@ unsafe fn emit_backtrace_by_frames(
     backtrace::trace_unsynchronized(|frame| {
         if resolve_frames == StacktraceCollection::EnabledWithInprocessSymbols {
             backtrace::resolve_frame_unsynchronized(frame, |symbol| {
+                if let Some(function) = symbol.name() {
```

Comment:
This would be better handled in the receiver.  We want to avoid as much as possible doing work in the collector